doc/Introduction.md

```diff
@@ -24,30 +32,61 @@ echo %VARIABLE% (Windows)
+| TAG                           | no       | master        | dc      | The tag defines the version of the image that is pulled from the docker hub.                                                                                                                                                                                                                                                                                                                    |
+| NODE_ENV                      | no       |               | ui      | If set to `development` search Trubudget's external services on localhost. If set to `production` disable Redux devtools extension                                                                                                                                                                                                                                                              |
+| REACT_APP_VERSION             | no       |               | ui      | Injected version via `$npm_package_version` in`.env` file to ensure the version is shown in the frontend                                                                                                                                                                                                                                                                                        |
+
+#### Kubernetes
+
+| Env Variable | Required | Default Value | Used by | Description                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| ------------ | -------- | ------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| EXPOSE_MC    | no       | false         | bc      | If set to true Trubudget tries to expose it's multichain via kubernetes' ingress. First a kubernetes config file is searched in `/.kube/config`. If not found and the MC is deployed in a cluster it is search for the service account to get the information. After configuration the `externalip` arg is set to the ip of the service' ingress of the configured clusterSERVICE_NAME and NAMESPACE are required to access the ingress of the service. |
```

Comment:
If not found and the MC is deployed in a cluster**, that it is searching** for the service account to get the information.